crt_portal/cts_forms/templates/forms/error_warning.html

```diff
@@ -1,5 +1,7 @@
         <div class="usa-alert usa-alert--warning usa-alert--slim">
-          <div class="usa-alert__body"><em class="usa-alert__text">
-            {{ errors }}
-          </em></div>
+          <div class="usa-alert__body">
+            <em role="alert" class="usa-alert__text">
```

Comment:
Hey @LindsayYoung! In the example that you shared, the role is being used as a selector. I don't think it's possible to set `role` dynamically using CSS, just as (I think!) it isn't possible to set the `class` or `id` of an element dynamically. It would be confusing if you could use CSS to set `role`, `class`, or `id`, since CSS takes those as inputs to add styles — you could easily end up with some bad infinite loops! 

Let me know if this makes sense to you! I do appreciate the suggestion. And of course PRs are always welcome!

Reply:
Ok, we will just have to add them to all the alert text. Thanks for looking into it